contracts/plugins/trading/DutchTrade.sol

```diff
@@ -274,6 +281,37 @@ contract DutchTrade is ITrade, Versioned {
         assert(status == TradeStatus.CLOSED);
     }
 
+    /// Create a trusted fill as an alternative to direct bidding
+    /// @param targetFiller The trusted filler implementation to use (e.g., CowSwapFiller)
+    /// @param deploymentSalt Unique salt for deterministic deployment
+    /// @return filler The deployed trusted filler contract
+    function createTrustedFill(address targetFiller, bytes32 deploymentSalt)
```

Comment:
I think this can be called for free to DoS `bid()` calls

Reply:
Yes. `bid()` would not work after this is called because tokens leave the DutchTrade.

Reply:
Added the `closeTrustedFiller` modifier as a pre-hook to ensure `bid` can be called.